src/app/app.component.ts

```diff
@@ -88,38 +91,42 @@ export class AppComponent implements OnInit, OnDestroy {
   }
 
   loadCmsSeries(auth: HalDoc) {
-    auth.followItems('prx:series', {filters: 'v4', zoom: 'prx:distributions'}).subscribe((series: HalDoc[]) => {
-      if (series.length === 0) {
-        this.error = 'Looks like you don\'t have any podcasts.';
+    auth.followItems('prx:series',
+      {per: auth.count('prx:series'), filters: 'v4', zoom: 'prx:distributions'}).subscribe((series: HalDoc[]) => {
+      if (series && series.length === 0) {
+        this.store.dispatch(new CmsPodcastsAction({podcasts: []}));
+      } else {
+        const podcasts: PodcastModel[] = series.map(doc => {
+          return {
+            doc,
+            seriesId: doc['id'],
+            title: doc['title']
+          };
+        });
+        const distros$ = podcasts.map(p => this.getSeriesPodcastDistribution(p));
+        Observable.zip(...distros$).subscribe(() => {
+          this.store.dispatch(new CmsPodcastsAction({podcasts: podcasts.filter(p => p.feederId)}));
```

Comment:
I either saw a podcast without a feederId or an episode without a guid, can't remember for sure. I'm having a hard time remembering, but an issue came up about the series or episode right around the same time. Figured it was better safe than sorry to do metrics for the good data and just skip over data problems it can't handle.

Reply:
👍